packages/client/package.json

```diff
@@ -86,7 +86,6 @@
     "react-dom": "18.3.1",
     "react-icons": "^4.2.0",
     "react-measure": "2.5.2",
-    "react-router": "6.10.0",
```

Comment:
apparently we only need to reference `react-router-dom` now .. updated remaining imports around the codebase